src/ansys/fluent/core/quantity.py

```diff
@@ -0,0 +1,553 @@
+from collections import OrderedDict
+
+
+class _UnitsTable(object):
+    fundamental_units = {
+        "kg": "M",
+        "g": "M",
+        "lb": "M",
+        "lbm": "M",
+        "slug": "M",
+        "m": "L",
+        "cm": "L",
+        "ft": "L",
+        "in": "L",
+        "s": "T",
+        "A": "I",
+        "K": "Temp",
+        "C": "Temp",
+        "F": "Temp",
+        "R": "Temp",
+        "mol": "N",
+        "cd": "J",
+        "sr": "SAngle",
+        "radian": "Angle",
+        "degree": "Angle",
+    }
+
+    derived_units = {
+        "N": "kg m s^-2",
+        "Pa": "N m^-2",
+        "W": "N m s^-1",
+        "J": "N m",
+        "V": "A ohm",
+        "F": "N m V^-2",
+        "H": "N m A^-2",
+        "S": "ohm^-1",
+        "Wb": "N m A^-1",
+        "T": "Wb m^-2",
+        "dyne": "g cm s^-2",
+        "erg": "dyne cm",
+        "pdl": "lbm ft s^-2",
+        "BTU": "J",
+        "psi": "lbf in^-2",
+        "lbf": "slug ft s^-2",
+        "psf": "lbf ft^-2",
+        "ohm": "kg m^2 s^-3 A^-2",
+        "Hz": "s^-1",
+    }
+
+    derived_units_with_conversion_factor = {
+        "l": (0.001, "m^3"),
+        "gal": (0.0037854117839999993, "m^3"),
+        "BTU": (1055.056, "J"),
+    }
+
+    multipliers = {
+        "d": 10**-1,
+        "c": 10**-2,
+        "m": 10**-3,
+        "μ": 10**-6,
```

Comment:
Do we support this character?

Reply:
It's ASCII character right ? 

`ALT + 230`

![image](https://user-images.githubusercontent.com/106588300/199182838-51c78e0f-4967-40bf-abed-7bdb92d16033.png)

Reply:
Probably it's better to use unicode: 

https://en.wikipedia.org/wiki/List_of_Unicode_characters#Greek_and_Coptic

Mu seems to be 956.

I think the ASCII characters beyond 128 depend on the loaded character set which is locale and maybe platform dependent.

Reply:
I am not able to load this file in Python.
```
e:\temp>c:\Python310\python.exe quantity.py
SyntaxError: Non-UTF-8 code starting with '\xb5' in file e:\temp\quantity.py on line 62, but no encoding declared; see https://python.org/dev/peps/pep-0263/ for details
```
I think the following line needs to be added at the top of the file:
# -*- coding: latin-1 -*-

Reply:
We have added `# -*- coding: latin-1 -*-` at the top of the file.

Reply:
Updated mu symbol by copying from character map.

![image](https://user-images.githubusercontent.com/106588300/199459785-c58a772e-93e1-4a8b-a0ff-7795e228046e.png)